browser/src/Services/Explorer/ExplorerSplit.tsx

```diff
@@ -74,12 +78,60 @@ export class ExplorerSplit {
         this._store.dispatch({ type: "LEAVE" })
     }
 
+    public moveFileOrFolder = (source: Node, dest: Node): void => {
```

Comment:
I like that this is decoupled from the UI and primarily dependent on the state/selectors - means we could have some other ways to implement this behavior. It'd be cool to hook up a `yank` + `paste` workflow later on!